### ISSUE TITLE
Hardcoded path to Xcode in relinkswiftstdlib.sh

### ISSUE THREAD

**Issue by whoover** [zhuowei/MarzipanTool#6]

**Comment by zhuowei** [comment]
Yeah - it made sense when this only worked with Xcode beta! I don't think this tool works anymore with newer Marzipan, anyways - try using [stroughtonsmith's Marzipanify](https://github.com/steventroughtonsmith/marzipanify) instead.
